addons/apecloud-postgresql/templates/_helpers.tpl

```diff
@@ -42,20 +49,277 @@ Generate scripts configmap
 {{- end }}
 {{- end }}
 
+{{/*
+Define apecloud postgresql 14 component definition name prefix
+*/}}
+{{- define "apecloud-postgresql14.componentDefNamePrefix" -}}
+{{- if eq (len .Values.cmpdVersionPrefix.apecloudPostgresql14) 0 -}}
+{{- printf "apecloud-postgresql14-" -}}
+{{- else -}}
+{{- printf "%s-" .Values.cmpdVersionPrefix.apecloudPostgresql14 -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Define apecloud postgresql 14 component configuration template name
+*/}}
+{{- define "apecloud-postgresql14.configurationTemplate" -}}
+apecloud-postgresql14-configuration-{{ .Chart.Version }}
+{{- end -}}
+
+{{/*
+Define apecloud postgresql 14 component config constraint name
+*/}}
+{{- define "apecloud-postgresql14.configConstraint" -}}
+apecloud-postgresql14-cc-{{ .Chart.Version }}
+{{- end -}}
+
 {{/*
 Define apecloud-postgresql component definition name prefix
 */}}
 {{- define "apecloud-postgresql.componentDefNamePrefix" -}}
-{{- printf "apecloud-postgresql-" -}}
+{{- printf "apecloud-postgresql" -}}
 {{- end -}}
 
 {{/*
 Define apecloud-postgresql14 component definition name
 */}}
 {{- define "apecloud-postgresql.compDefApecloudPostgresql14" -}}
 {{- if eq (len .Values.resourceNamePrefix) 0 -}}
```

Comment:
remove resourceNamePrefix, it's unnecessary

Reply:
done